source/apps/configure/config.txt

```diff
@@ -0,0 +1,124 @@
+.. _app-configuration:
+
+=================
+App Configuration
+=================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. toctree::
+   
+<<<<<<< HEAD:source/reference/config.txt
+   App Services App </reference/config/app>
+   Users & Authentication Providers </reference/config/auth>
+   MongoDB Data Sources </reference/config/data_sources>
+   Environment Values </reference/config/environments>
+   Functions </reference/config/functions>
+   GraphQL </reference/config/graphql>
+   Static Hosting </reference/config/hosting>
+   HTTP Endpoints </reference/config/http_endpoints>
+   Log Forwarders </reference/config/log_forwarders>
+   Third-Party Services </reference/config/services>
+   Atlas Device Sync </reference/config/sync>
+   Triggers </reference/config/triggers>
+   Values </reference/config/values>
+   Create Template Configurations with Expansions </reference/config/template-expansions>
+   Legacy Configuration Files </reference/config/legacy>
+=======
```

Comment:
dis aint right

Reply:
Good catch - this page was supposed to be deleted entirely